Description: Update ccpdep.pl for GCC 4.8
 ccdep.pl was failing with GCC 4.8 error messages. This patch takes these new
 messages into account.
Author: Tanguy Ortolo <tanguy+debian@ortolo.eu>
Bug-Debian: http://bugs.debian.org/701351
Last-Update: 2013-08-13

--- sam2p-0.49.1.orig/ccdep.pl
+++ sam2p-0.49.1/ccdep.pl
@@ -231,6 +231,8 @@ while ($R=~/\G(.*)\n?/g) {
     undef $included_from;
   } elsif ($S eq 'compilation terminated.') {  # Useless message, ignore.
   } elsif ($S=~/^distcc\[/) {  # Useless message, ignore.
+  } elsif ($S=~/^ *#/) {  # Useless message, ignore.
+  } elsif ($S=~/^ *\^/) {  # Useless message, ignore.
   } else {
     die "$0: invalid depret: [$S]\n";
   }
